core/PRP/Physics/plGenericPhysical.cpp

```diff
@@ -717,7 +741,13 @@ void plGenericPhysical::IWritePXPhysical(hsStream* S, plResManager* mgr) {
         if (!NxCookTriangleMesh(triDesc, buf))
             throw hsBadParamException(__FILE__, __LINE__, "Incorrect data for a PhysX Trimesh Bake");
 #else
-        throw hsNotImplementedException(__FILE__, __LINE__, "PhysX TriangleMesh");
+        S->write(4, "HSP\x01");
+        S->writeInt(fVerts.size());
+        for (size_t i = 0; i < fVerts.size(); ++i)
+            fVerts[i].write(S);
+        S->writeInt(fIndices.size() / 3);
```

Comment:
Should we add a check somewhere that the indices array's size is valid (`%3 == 0`)?